docs/howto/garbage-collection/standalone-gc.md

```diff
@@ -18,67 +18,59 @@ experimental
 {: .label .label-red }
 
 {: .note }
-> Standalone GC is only available for [lakeFS Enterprise]({% link enterprise/index.md %}). \
-Please verify with your account manager that your license includes this feature.
-
+> Standalone GC is only available for [lakeFS Enterprise]({% link enterprise/index.md %}).
 
 {: .note .warning }
-> Standalone GC is experimental and offers limited capabilities compared to the [Spark-backed GC]({% link howto/garbage-collection/gc.md %}). Read through the [limitations](./standalone-gc.md#limitations) carefully before using it.
+> Standalone GC is experimental and offers limited capabilities compared to the [Spark-backed GC]({% link howto/garbage-collection/gc.md %}). For large scale environments, we recommend using the Spark-backed solution.
 
-{% include toc_2-4.html %}
+{% include toc_2-3.html %}
 
-## About
+## What is Standalone GC?
 
-Standalone GC is a limited version of the Spark-backed GC that runs without any external dependencies, as a standalone docker image.
+Standalone GC is a simplified version of the Spark-backed GC that runs without any external dependencies, delivered as a standalone
+docker image. It supports S3 and [self-managed S3 compatible storages](#s3-compatible-clients) such as MinIO.    
 
 ## Limitations
 
-1. Except for the [Lab tests](./standalone-gc.md#lab-tests) performed, there are no further guarantees about the performance profile of the Standalone GC. 
-2. Horizontal scale is not supported - Only a single instance of `lakefs-sgc` can operate at a time on a given repository.
-3. Standalone GC only marks objects and does not delete them - Equivalent to the GC's [mark only mode]({% link howto/garbage-collection/gc.md %}#mark-only-mode). \
-   More about that in the [Get the List of Objects Marked for Deletion](./standalone-gc.md#get-the-list-of-objects-marked-for-deletion) section.
-
-### Lab tests
+1. **No horizontal scalability**: Only a single instance of `lakefs-sgc` can operate on a given repository at a time.
```

Comment:
Now that I think about it - another limitation compared to the spark-backed GC is that this one only supports AWS (and s3 compatible) storage? the spark-backed one supports any blockstore, right?

Reply:
added a bullet